client/src/components/MapPopupContent.vue

```diff
@@ -64,9 +64,24 @@ export default defineComponent({
       this.displayedProperties.clear();
 
       for (let entry of this.featureProperties) {
+        console.log(this.featureProperties);
```

Comment:
If this log is intentional, can you add some string before it to put it into context?

Reply:
Whoops good catch thanks!